tools/c7n_org/c7n_org/cli.py

```diff
@@ -458,6 +458,599 @@ def report(config, output, use, output_dir, accounts,
     writer.writerows(rows)
 
 
+def validate_basic(custodian_config, policy_file, fmt, check_mode, verbose):
+    """Run basic account-agnostic validation (Phase 1 logic).
+
+    Args:
+        custodian_config: Parsed policy configuration
+        policy_file: Path to policy file (for error reporting)
+        fmt: File format ('yml', 'yaml', 'json')
+        check_mode: Deprecation check mode (deprecated.SKIP, deprecated.STRICT, or None)
+        verbose: Verbose logging flag
+
+    Returns:
+        bool: True if validation passes, False otherwise
+    """
+    from c7n import schema, deprecated
+    from c7n.schema import StructureParser
+    from c7n.policy import Policy, PolicyValidationError
+    from c7n.config import Config, Bag
+    from c7n.resources import load_resources
+    from c7n.loader import SourceLocator
+
+    # Core validation logic
+    structure = StructureParser()
+    errors = []
+    used_policy_names = set()
+    found_deprecations = False
+    footnotes = deprecated.Footnotes()
+
+    # Structure validation
+    log.debug("Running structure validation")
+    try:
+        structure.validate(custodian_config)
+    except PolicyValidationError as e:
+        log.error(f"Configuration invalid: {policy_file}")
+        log.error(str(e))
```

Comment:
Maybe `log.exception(e)` instead? Unless this is for consistency with how things are done elsewhere in the file...